src/main/java/com/google/sps/task/Task.java

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.task;
+
+public final class Task {
```

Comment:
It would be helpful to add a method in this class that understands how to make a Task entity from Datastore into an instance of this class. You can do this by adding another constructor that has a different signature, namely a Datastore entity instance. Then you don't have to update a bunch of call sites whenever you add a field to this model.

Reply:
Ahh that really helps for entities with so many fields. I would do this in the new commit by adding a new constructor that takes in an Entity and outputs a task instance.